### PR TITLE
add for Realme 8 5G / Narzo 30 5G

### PR DESCRIPTION
**Pull request by crazo7924**

### CODE REVIEW

**Comment by crazo7924** [comment]
I was dumb enough to tag in my own repo

File: Realme/oppo6833-SystemUI/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.realme.oppo6833.systemui"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="com.android.systemui"
+                android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
+                android:requiredSystemPropertyValue="+oppo6833*"
```
Comment (phhusson) [inline Realme/oppo6833-SystemUI/AndroidManifest.xml]: pretty sure that's wrong

File: Realme/RE513CL1-SystemUI/res/values/dimens.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+-->
+<resources>
+    <dimen name="status_bar_height_landscape">110.0px</dimen>
```
Comment (phhusson) [inline Realme/RE513CL1-SystemUI/res/values/dimens.xml]: You shouldn't override those from SystemUI overlay, but from main overlay

File: Realme/RE513CL1/res/values/notch.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Height of the status bar in portrait -->
+    <dimen name="status_bar_height_landscape">110.0px</dimen>
+    
+    <dimen name="quick_qs_offset_height">@dimen/status_bar_height_portrait</dimen>
+
+    <!-- Height of the status bar in landscape -->
+    <dimen name="status_bar_height_portrait">110.0px</dimen>
```
Comment (phhusson) [inline Realme/RE513CL1/res/values/notch.xml:7]: It sounds pretty weird to have the same in  landscape and portrait, got photos?

Comment (crazo7924) [reply Realme/RE513CL1/res/values/notch.xml:7]: This device has a punch hole display.
I just used whatever values the overlay apks had

File: Realme/RE513CL1-SystemUI/res/values/dimens.xml (hunk 2 above)
Comment (crazo7924) [reply Realme/RE513CL1-SystemUI/res/values/dimens.xml]: Ok 👍

File: Realme/oppo6833-SystemUI/AndroidManifest.xml (hunk 1 above)
Comment (crazo7924) [reply Realme/oppo6833-SystemUI/AndroidManifest.xml]: see 9f6692c49229f0e8effcc6a8ebcc099cce88922b

File: Realme/RE513CL1/res/xml/power_profile.xml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    <array name="cpu.core_power.cluster1">
+        <value>25</value>
+        <value>35</value>
+        <value>50</value>
+        <value>60</value>
+    </array>
+    <array name="memory.bandwidths">
+        <value>22.7</value>
+    </array>
+    <item name="battery.capacity">1000</item>
```
Comment (phhusson) [inline Realme/RE513CL1/res/xml/power_profile.xml]: Sounds wrong to me...

File: Realme/RE513CL1/res/xml/power_profile.xml [hunk 5]
```diff
@@ -1,36 +1,12 @@
...
-    <item name="camera.flashlight">0.1</item>
-    <item name="camera.avg">0.1</item>
-    <item name="gps.on">0.1</item>
-    <item name="radio.active">0.1</item>
-    <item name="radio.scanning">0.1</item>
-    <array name="radio.on">
-        <value>0.2</value>
-        <value>0.1</value>
-    </array>
+    <item name="battery.capacity">3000</item>
```
Comment (phhusson) [inline Realme/RE513CL1/res/xml/power_profile.xml]: https://www.gsmarena.com/realme_8_5g-10846.php says this is wrong

Comment (crazo7924) [reply Realme/RE513CL1/res/xml/power_profile.xml]:
```
:/ # cat /sys/class/power_supply/battery/charge_full
4000
:/ #
```
it's actually 4000 mAh and not 5000 or 3000 😐

**Comment by phhusson** [comment]
I doubt just changing the battery capacity is enough, I'm expecting that all the other values are wrong as well.

You need to take the power_profile.xml "from the right place". I can recommend you to run https://github.com/phhusson/OverlayMaker on stock rom, it should dump power_profile from rom in /sdcard/Android/data/me.phh.overlaymaker/cache

**Comment by crazo7924** [comment]
> I doubt just changing the battery capacity is enough, I'm expecting that all the other values are wrong as well.
> 
> You need to take the power_profile.xml "from the right place". I can recommend you to run https://github.com/phhusson/OverlayMaker on stock rom, it should dump power_profile from rom in /sdcard/Android/data/me.phh.overlaymaker/cache

I got placeholder values for `power_profile.xml` from your OverlayMaker app

```
<?xml version="1.0" encoding="utf-8"?>
<device name="Android"> <item name="ambient.on">0.1</item>
 <item name="screen.on">0.1</item>
 <item name="screen.full">0.1</item>
 <item name="bluetooth.active">0.1</item>
 <item name="bluetooth.on">0.1</item>
 <item name="wifi.on">0.1</item>
 <item name="wifi.active">0.1</item>
 <item name="wifi.scan">0.1</item>
 <item name="audio">0.1</item>
 <item name="video">0.1</item>
 <item name="camera.flashlight">0.1</item>
 <item name="camera.avg">0.1</item>
 <item name="gps.on">0.1</item>
 <item name="radio.active">0.1</item>
 <item name="radio.scanning">0.1</item>
 <array name="radio.on">  <value>0.2</value>
  <value>0.1</value>
</array>
 <array name="cpu.clusters.cores">  <value>4</value>
  <value>4</value>
</array>
 <array name="cpu.speeds.cluster0">  <value>400000</value>
</array>
 <array name="cpu.active.cluster0">  <value>0.1</value>
</array>
 <item name="cpu.idle">1.11</item>
 <item name="cpu.suspend">5</item>
 <item name="cpu.active">2.55</item>
.  <item name="cpu.cluster_power.cluster0">2.11</item>
 <item name="cpu.cluster_power.cluster1">2.22</item>
 <array name="cpu.core_speeds.cluster0">  <value>300000</value>
  <value>1000000</value>
  <value>2000000</value>
</array>
 <array name="cpu.core_speeds.cluster1">  <value>300000</value>
  <value>1000000</value>
  <value>2500000</value>
  <value>3000000</value>
</array>
 <array name="cpu.core_power.cluster0">  <value>10</value>
  <value>20</value>
  <value>30</value>
</array>
 <array name="cpu.core_power.cluster1">  <value>25</value>
  <value>35</value>
  <value>50</value>
  <value>60</value>
</array>
 <array name="memory.bandwidths">  <value>22.7</value>
</array>
 <item name="battery.capacity">1000</item>
 <item name="wifi.controller.idle">0</item>
 <item name="wifi.controller.rx">0</item>
 <item name="wifi.controller.tx">0</item>
 <array name="wifi.controller.tx_levels"></array>
 <item name="wifi.controller.voltage">0</item>
 <array name="wifi.batchedscan">  <value>.0002</value>
  <value>.002</value>
  <value>.02</value>
  <value>.2</value>
  <value>2</value>
</array>
 <item name="modem.controller.sleep">0</item>
 <item name="modem.controller.idle">0</item>
 <item name="modem.controller.rx">0</item>
 <array name="modem.controller.tx">  <value>0</value>
  <value>0</value>
  <value>0</value>
  <value>0</value>
  <value>0</value>
</array>
 <item name="modem.controller.voltage">0</item>
 <array name="gps.signalqualitybased">  <value>0</value>
  <value>0</value>
</array>
 <item name="gps.voltage">0</item>
</device>
```

**Pull request merged**